charts/argo-cd/values.yaml

```diff
@@ -248,6 +248,8 @@ configs:
     controller.self.heal.timeout.seconds: 5
     # -- Repo server RPC call timeout seconds.
     controller.repo.server.timeout.seconds: 60
+    # -- Sync wave delay seconds to make delay between applications' sync
+    controller.sync.wave.delay: 2
```

Comment:
📝 default is `2` 
https://argo-cd.readthedocs.io/en/stable/user-guide/sync-waves/
<img width="751" alt="image" src="https://github.com/user-attachments/assets/9e0163ad-e177-4dec-9a62-3e1c0aaaca04">